ED/src/init/ed_params.f90

```diff
@@ -3536,9 +3528,19 @@ subroutine init_pft_alloc_params()
             !    Amazon.  Global Biogeochem. Cycles, 30(11):1639-1660.                     !
             !    doi:10.1002/2016GB005465 (L16).                                           !
             !------------------------------------------------------------------------------!
-            b1Bl(ipft) = c14f15_bl_xx(1) / SLA(ipft)
+            b1Bl(ipft) = c14f15_bl_xx(1) / SLA(ipft) ! XX --> MLO: should ther be a C2B here given c14f15_bl_xx is in m2 (?) and SLA is m2/kgC
```

Comment:
@xiangtaoxu I need to double check this, but I think I took this into account. The C2B is applied in allometry_utils.f90, to be consistent with the previous allometric equations.